modular_skyrat/modules/food_replicator/replicator_designs/replicator_food.dm

```diff
@@ -0,0 +1,111 @@
+	id = "mexican_chips"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/food/colonial_course/nachos
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/blins
+	name = "foreign colonization ration, dessert"
+	id = "slavic_crepes"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/food/colonial_course/blins
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/spork
+	name = "foreign colonization ration, utensils"
+	id = "slavic_utens"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/storage/box/utensil
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/fasani
+	name = "foreign colonization ration, bubblegum pack"
+	id = "slavic_gum"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/storage/box/gum/colonial
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/cup
+	name = "paper cup, stylised, empty"
+	id = "slavic_cup"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 10)
+	build_path = /obj/item/reagent_containers/cup/glass/coffee/colonial/empty
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/tea
+	name = "black tea, powdered"
+	id = "slavic_tea"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 4)
+	make_reagent = /datum/reagent/consumable/powdered_tea
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/coffee
+	name = "coffee, powdered"
+	id = "slavic_covfefe"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 4)
+	make_reagent = /datum/reagent/consumable/powdered_coffee
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/cacoca
+	name = "hot chocolate, powdered"
+	id = "slavic_coco"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 4)
+	make_reagent = /datum/reagent/consumable/powdered_coco
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/lemonade
+	name = "lemonade, powdered"
+	id = "slavic_lemon"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 4)
+	make_reagent = /datum/reagent/consumable/powdered_lemonade
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/sugar
+	name = "sugar"
+	id = "slavic_sugar"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 5)
+	make_reagent = /datum/reagent/consumable/sugar
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/milk
+	name = "milk, powdered"
+	id = "slavic_milk"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 4)
+	make_reagent = /datum/reagent/consumable/powdered_milk
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
```

Comment:
I still don't really understand why we'd have powdered milk, when there's already a perfectly valid milk option in regular biogenerators. This one feels particularly useless.

Also, please create unique designs, with this you're overwriting the standard milk and sugar designs, at the very least.

Reply:
Immersion; it honestly feels better when your food has unique, semi-realistic interactions instead of it being incredibly simplified.
*also oops didn't notice that.*